src/excmds.ts

```diff
@@ -3484,6 +3485,11 @@ export async function hint(option?: string, selectors?: string, ...rest: string[
         option = "-" + option.slice(2)
         rapid = true
     }
+    let jshints = true
+    if (option.startsWith("-j")) {
```

Comment:
My reading of this code is that `-qjc` would trigger the rapid option but the jshints option; `-jqc` would trigger jshints but not rapid.

"preclude" is just a fancy way of saying "stop".

Reply:
You are right, I didn't realize that we are not looping though the options as in bash, my bad, I will fix this.